tests/test_routes/test_main_routes.py

```
@@ -0,0 +1,217 @@
+        # メトリクスモジュールが利用可能な場合は200、そうでなければ503
+        assert response.status_code in [200, 503]
+
+    def test_メトリクスモジュールがない場合503(self, client):
+        """メトリクスモジュールがない場合は503を返す"""
+        with patch("utils.performance.get_metrics") as mock_metrics:
+            # ImportErrorを発生させる
+            mock_metrics.side_effect = ImportError("Module not found")
+
+            response = client.get("/api/metrics")
+
+            # 503またはエラー処理（実装によって異なる）
```

Ruff (0.14.6)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)